Source/Core/DolphinQt2/TAS/Shared.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2018 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+#include <QWidget>
+
+class QDialog;
+class QString;
+class QSpinBox;
+class QCheckBox;
+class QBoxLayout;
+class QGroupBox;
+struct GCPadStatus;
+
+QGroupBox* CreateStickInputs(QDialog* window, QString name, QSpinBox** x_value, QSpinBox** y_value,
```

Comment:
Maybe use references instead of pointers? (e.g. ``QSpinBox*&``)

Reply:
done